spec/services/ants_api_spec.rb

```diff
@@ -1,7 +1,7 @@
-RSpec.describe AntsApi::Appointment, type: :service do
+RSpec.describe AntsApi, type: :service do
   include_context "rdv_mairie_api_authentication"
 
-  describe ".find_by" do
+  describe "#status" do
```

Comment:
Je chipote, mais on décrit bien des méthodes de classe ici, non ?

```suggestion
  describe ".status" do
```

Reply:
Corrigé ici: aefdc54493f71a6350e4e0884ac572002584ce4c